server/views/pages/goal/add-note/index.njk

```diff
@@ -35,7 +32,7 @@
             id: "note",
             value: form.note,
             label: {
-              text: "Add a note to this goal (optional)",
```

Comment:
Just playing devil's advocate a little here ... I get the whole re-use thing, but to me this feels a little hidden. It feels like the question text has a hidden coupling with the html page title 🤔 

Could there be a risk that a naive dev (me or @mattwills-moj-digital ) could come along in the future and change the page title variable (because we'd been asked to change the page title), but not realise we'd also changed the question text?

What do you think?